3.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tailwind</title>
+</head>
+<body>
+    <header class="header">
+        <div class="header__left-column">
+            <h2 class="header__left-column__h2">Press</h2>
+            <p class="header__left-column__text">Et malesuada fames ac turpis egestas sed tempus. </p>
+        </div>
+        <div class="header__right-column">
+            <form action="GET" class="header__right-column form">
+                <input type="text" placeholder="Enter your name" class="form__input">
+                <button type="submit" class="form__button">Notify me</button>
+            </form>
+        </div>
+    </header>
+    <div class="main-content">
+        <article class="main-content__element">
+            <time class="dtstart" datetime="2005-10-05" class="main-content__element__time">October 5</time>
```

Comment:
```suggestion
            <time class="dtstart" datetime="2005-10-05" class="main-content__element__time">October 5</time>
```
не валидно, 2 класса, и также не нужно писать main-content__element__time
если хочется так написать, то разбивайте